ios/Classes/Bindings/VideoPresentationServiceBinding.swift

```diff
@@ -19,7 +19,7 @@ class VideoPresentationServiceBinding: Binding {
     
     override init(name: String, registrar: FlutterPluginRegistrar) {
         super.init(name: name, registrar: registrar)
-        VoxeetSDK.shared.videoPresentation.delegate = self
+        // VoxeetSDK.shared.videoPresentation.delegate = self
```

Comment:
Why is this commented out?

Reply:
I think that's a mistake. I'll revert that.